### PR TITLE
Use redeployed CCTP contracts

### PR DESCRIPTION
**Pull request by nambrot**
### Description

Circle updated the contracts

### CODE REVIEW

File: typescript/sdk/src/deploy/middleware/LiquidityLayerApp.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ const CIRCLE_ATTESTATIONS_BASE_URL =
 
 const PORTAL_VAA_SERVICE_SUCCESS_CODE = 5;
 
-const CircleBridgeInterface = ICircleBridge__factory.createInterface();
+const CircleBridgeInterface = ITokenMessenger__factory.createInterface();
```
Comment (tkporter) [inline typescript/sdk/src/deploy/middleware/LiquidityLayerApp.ts:28]: Think it's worth renaming this too?

**Review by tkporter — APPROVED** [review]

**Comment by asaj** [comment]
Need to update the testnet3 addresses

File: typescript/infra/config/environments/testnet2/middleware/liquidity-layer/addresses.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
   "fuji": {
-    "circleBridgeAdapter": "0xb54AD7AE42B7c505100594365CdBC4b28Ef51FE6",
+    "circleBridgeAdapter": "0x17EB33454AAEF8E91510540a0ebF4a8213dd740D",
```
Comment (asaj) [inline typescript/infra/config/environments/testnet2/middleware/liquidity-layer/addresses.json:3]: Does this PR include a deployment of the liquidity layer? I assume not?

File: typescript/infra/config/environments/testnet2/middleware/liquidity-layer/verification.json [hunk 3]
```diff
@@ -17,6 +17,18 @@
       "address": "0xe589a05be1304b43A6FEb9c5D6a6EEEA35656271",
       "isProxy": false,
       "constructorArguments": ""
+    },
```
Comment (asaj) [inline typescript/infra/config/environments/testnet2/middleware/liquidity-layer/verification.json:20]: Wondering why this is here, unless it *does* include a deployment? If it includes a deployment wouldn't we expect all of the addresses to change?

**Comment by asaj** [comment]
Oops I meant to be reviewing #1497

**Pull request merged**